README.md

```diff
@@ -1,82 +1,91 @@
-# spark-rock
-ROCK for Spark on Kubernetes. It provides a direct port of this Spark provided [Dockerfile](https://github.com/apache/spark/blob/master/resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile).
+## Introduction to Charmed MongoDB ROCK  (OCI Image)
 
-## Build the image yourself
+Charmed Spark is a set of Canonical supported artifacts (including charms, ROCK OCI images and SNAPs) that makes operating Spark workloads on Kubernetes seamless, secure and production-ready. 
 
-After cloning the repo, run the following commands.
+The solution helps to simplify user interaction with Spark applications and the underlying Kubernetes cluster whilst retaining the traditional semantics and command line tooling that users already know. Operators benefit from straightforward, automated deployment of Spark components (e.g. Spark History Server) to the Kubernetes cluster, using [Juju](https://juju.is/). 
 
-- Build the Spark ROCK.
-```bash
-sudo snap install rockcraft --classic --edge
-rockcraft pack --verbose
+Deploying Spark applications to Kubernetes has several benefits over other cluster resource managers such as Apache YARN, as it greatly simplifies deployment, operation, authentication while allowing for flexibility and scaling. However, it requires knowledge on Kubernetes, networking and coordination between the different components of the Spark ecosystem in order to provide a scalable, secure and production-ready environment. As a consequence, this can significantly increase complexity for the end user and administrators, as a number of parameters need to be configured and prerequisites must be met for the application to deploy correctly or for using the Spark CLI interface (e.g. pyspark and spark-shell). 
+
+Charmed Spark helps to address these usability concerns and provides a consistent management interface for operations engineers and cluster administrators who need to manage enablers like Spark History Server.
+
+### Features 
+
+The Charmed Spark Rock comes with some build-in tooling embedded:
+
+* Canonical-supported Spark binaries 
+* [`spark8t`](https://github.com/canonical/spark-k8s-toolkit-py) CLI for managing Spark service accounts
+* [Pebble](https://github.com/canonical/pebble) managed services:
+  * [Spark History Server](https://spark.apache.org/docs/latest/monitoring.html) for monitoring your Spark jobs
+
+## Version
+
+ROCKs will be named as `<version>-<series>_<risk>`.
+
+`<version>` is the software version; `<series>` is the Ubuntu LTS series that ROCKs supports; and the <risk> is the type of release, if it is edge, candidate or stable. Example versioning will be 5-22.04_stable which means Charmed MongoDB is a version 5 of the software, supporting the 22.04 Ubuntu release and currently a 'stable' version of the software. See  versioning details [here](https://snapcraft.io/docs/channels).
```

Comment:
Please change the example, it is still mentioning 'MongoDB'
Is it the snap versioning mentioned on purpose ?

Reply:
Yes, the SNAP is fine. It is the post where the concepts of channels/track/risk is explained